src/opnsense/scripts/systemhealth/queryLog.py

```diff
@@ -68,7 +68,7 @@
             filenames = glob.glob("%s/%s_*.log" % (log_basename, log_basename.split('/')[-1].split('.')[0]))
             for filename in sorted(filenames, reverse=True):
                 log_filenames.append(filename)
-        # legacy log output is always stiched last
+        # legacy log output is always stitched last
```

Comment:
```suggestion
        # legacy log output is always stiched last
```

leave out, dead code needs removal